zookeeper-server/src/main/java/org/apache/zookeeper/server/persistence/FileTxnLog.java

```diff
@@ -376,6 +387,12 @@ public long getLastLoggedZxid() {
     public synchronized void commit() throws IOException {
         if (logStream != null) {
             logStream.flush();
+            filePosition += unFlushedSize;
+            //It is the same as the FilePadding.calculateFileSizeWithPadding line_106.
```

Comment:
unfortunately "line_106" will move in the future, please remove this word.

you can add a javadoc with a {@link } comment to the method

Reply:
addressed.